README.md

```diff
@@ -103,16 +101,16 @@ Jump into our <a href="http://slack.layer5.io"><img src=".github/assets/images/s
 <a href="http://slack.layer5.io"><img alt="Layer5 Service Mesh Community" src=".github/assets/images/buttons/community.png" style="margin-right:10px;" width="125px" align="left" /></a>
 ✔️ <em><strong>Join</strong></em> any or all of the weekly meetings on the <a href="https://calender.google.com/calendar/b/1?cid=bGF5ZXI1LmlvX2VoMmFhOWRwZjFnNDBlbHZvYzc2MmpucGhzQGdyb3VwLmNhbGVuZGFyLmdvb2dsZS5jb20">community calendar</a>.<br />
 ✔️ <em><strong>Watch</strong></em> community <a href="https://www.youtube.com/playlist?list=PL3A-A6hPO2IMPPqVjuzgqNU5xwnFFn3n0">meeting recordings</a>.<br />
-✔️ <em><strong>To Access </strong></em><a href="https://drive.google.com/drive/u/4/folders/0ABH8aabN4WAKUk9PVA">community drive, fill Community Member Form</a>.<br />
+✔️ <em><strong>To Access community drive,</strong></em><a href="https://drive.google.com/drive/u/4/folders/0ABH8aabN4WAKUk9PVA"> fill Community Member Form</a>.<br />
 <br />
 </p>
 <p>
 <i>Not sure where to start?</i> Grab an open issue with the <a href="https://github.com/issues?utf8=✓&q=is%3Aopen+is%3Aissue+archived%3Afalse+org%3Alayer5io+label%3A%22help+wanted%22+">help-wanted label</a>
 </p>
 <p>Find us on Twitter: <a href="https://twitter.com/layer5"><img src=".github/assets/images/twitter.png" height="16px"
```

Comment:
Put the image before ":". So that it's common for all of them.

Reply:
could you pls explain a bit, Sorry I didn't get you.

Reply:
Make this
```
<p>Find us on Twitter: <a href="https://twitter.com/layer5"><img src=".github/assets/images/twitter.png" height="16px" align="bottom />
```
like this
```
<p>Find us on Twitter <img src=".github/assets/images/twitter.png" height="16px" align="bottom" />:```
And put the link only around the `@layer5` text.

Reply:
thank you got it!

Reply:
Gents, Could we make line 104 match verbatim the same line in https://github.com/layer5io/layer5-repo-template‘s readme?

Reply:
Yeah, that would be better.

Reply:
ok on it

Reply:
@leecalcote  I have done the requested changes pls see to it.